shared/constants/whats-new.tsx

```diff
@@ -0,0 +1,68 @@
+import semver from 'semver'
+import {
+  NoVersion,
+  CurrentVersion,
+  LastVersion,
+  LastLastVersion,
+  WhatsNewVersion,
+  WhatsNewVersions,
+} from './types/whats-new'
+
+/*
+ * IMPORTANT:
+ *    1. currentVersion > lastVersion > lastLastVersion
+ *    2. Must be semver compatible
+ *    Source: https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string
+ *
+ * HOW TO ADD A NEW RELEASE
+ *    1. lastLastVersion = lastVersion
```

Comment:
`lastLastVersion` is the version before last?  I wonder if `versionBeforeLast` would be better.